subprojects/docs/src/docs/release/notes.md

```diff
@@ -36,6 +36,28 @@ As of Gradle 4.4, optional dependencies will participate in dependency resolutio
 For example, if a transitive dependency on `foo:bar:1.1` is optional, but another path in the dependency graph brings `foo:bar:1.0` (not optional), then Gradle will resolve to `foo:bar:1.1`.
 Previous releases would resolve to `foo:bar:1.0`. However, if no "hard" dependency is found on the optional module, then it will **not** be included, as previous Gradle versions did.
 
+### Support for importing Maven Bill of Materials (BOMs)
+
+Building on top of the new optional dependencies support, Gradle now supports the import of BOMs by declaring them as dependencies.
```

Comment:
⭕️ It's just a coincidence that we implement them the same way. I wouldn't tell users about that, it would be confusing for Maven users I think.

Reply:
Good point. I removed that.